apluslms_roman/cli.py

```diff
@@ -842,3 +847,8 @@ def backend_test_action(context, verbose=False):
 
 if __name__ == '__main__':
     main()
+
+
+def backend_clean_action(context):
```

Comment:
Shouldn't be after `__name__ == '__main__'`.

Reply:
ah. that _was_ my intention, didn't notice that there :')